hydrus/parser/openapi_parser.py

```diff
@@ -268,26 +273,28 @@ def generateOrUpdateClass(name, collection, global_, path)->bool:
 
 def check_for_ref(global_: Dict[str, Any],
                   path: str,
-                  block: Dict[str, Any])->str:
+                  block: Dict[str, Any]) -> str:
     """
-    Checks for references in responses and parameters key , and adds classes to state
+    Checks for references in responses and parameters key ,
+    and adds classes to state
     :param global_: global state
     :param path: endpoint
     :param block: block containing specific part of doc
     :return: class name
     """
 
-    # will check if there is an external ref , go to that location , add the class in globals , will also verify
+    # will check if there is an external ref , go to that location,
+    # add the class in globals , will also verify
     # if we can parse this method or not , if all good will return class name
     for obj in block["responses"]:
         try:
             try:
                 # can only be internal
-                class_location = block["responses"][obj]["schema"]["$ref"].split(
-                    '/')
+                class_location = block["responses"][obj]["schema"]["$ref"].\
+                    split('/')
             except KeyError:
-                class_location = block["responses"][obj]["schema"]["items"]["$ref"].split(
-                    '/')
+                class_location = block["responses"][obj]["schema"]["items"]\
```

Comment:
Remove the whitespace at the end of this line

Reply:
That will not solve the issue. It will end up ignoring the code on next line.

Reply:
Keeping that line long will solve this issue : `class_location = block["responses"][obj]["schema"]["items"]["$ref"].\`
 but will end up violating the max-line-length constraint!

Reply:
Exactly the reason why it was made into 2 lines in the first place.

Reply:
Make it 3 lines just as you have done in line 438: 
`class_location = \
                    block["responses"][obj]["schema"]["items"]["$ref"].\
                    split('/')`

Reply:
Thanks!